src/forms/TextInput.jsx

```diff
@@ -5,79 +5,62 @@ import cx from 'classnames';
 /**
  * @module TextInput
  */
-class TextInput extends React.Component {
+function TextInput(props) {
```

Comment:
One thing to note with doing it this way (anonymous function) is that it can be trickier to figure out what the problem is if you are looking at an error's call stack. Personally, I prefer to reference [airbnb's style guide](https://github.com/airbnb/javascript/tree/master/react#class-vs-reactcreateclass-vs-stateless) where they recommend explicitly naming the function in these sort of situations. Not a major thing, but something to be aware of